spec.bs

```diff
@@ -2119,37 +2138,18 @@ They return an [=interest group=] or null:
 </div>
 
 <div algorithm>
-To <dfn>obtain the Private Aggregation coordinator</dfn> given a
+To <dfn lt="obtain the Private Aggregation coordinator from a PA config">obtain
+the Private Aggregation coordinator</dfn> given a
```

Comment:
nit: is there a reason to keep this the same name as the other? Is it slightly easier to just call it "... from a PA config"

Reply:
I was treating them like "overloaded" functions, but I don't feel strongly. That being said, I don't love the "PA config" if it's the proper name -- maybe we could call it "obtain the coordinator from a Private Aggregation config"?

Reply:
Went with this!

Reply:
LGTM